bilby/core/prior/conditional.py

```diff
@@ -324,38 +324,46 @@ class ConditionalInterped(conditional_prior_factory(Interped)):
 
 class DirichletElement(ConditionalBeta):
     r"""
-    Single element in a dirichlet distribution
-
-    The probability scales as
+    Single element in a dirichlet distribution. The probability defined as,
 
     .. math::
-        p(x_n) \propto (x_\max - x_n)^{(N - n - 2)}
+        p(x_n\mid S_n) := \begin{cases}
+            \displaystyle\frac{(N - n - 1)(1 - S_n - x_n)^{N - n - 2}}{(1-S_n)^{N-n-1}}
+            & n < N - 1 \\
+            \displaystyle\delta(1-S_{N-1}-x_{N-1}) & n = N - 1
+        \end{cases}, \qquad 0 \leq x_n \leq 1 - S_n
+
+    where, :math:`\delta` is the Dirac delta function, :math:`N` is the total number of
+    dimensions and :math:`S_n` is the sum of all previous dimensions,
 
-    for :math:`x_n < x_\max`, where :math:`x_\max` is the sum of :math:`x_i`
-    for :math:`i < n`
+    .. math::
+        S_n := \sum_{i=0}^{n-1}x_i & n < N
 
     Examples
     ========
-    n_dimensions = 1:
 
-        .. math::
-            p(x_0) \propto 1 ; 0 < x_0 < 1
+        :code:`n_dimensions` = :math:`N = 1`:
 
-    n_dimensions = 2:
-        .. math::
-            p(x_0) &\propto (1 - x_0) ; 0 < x_0 < 1
-            p(x_1) &\propto 1 ; 0 < x_1 < 1
+            .. math::
+                p(x_0) = 1 ; 0 < x_0 < 1
```

Comment:
I might be missing something, but following the equation about I think this should be:
```
p(x_0) = \delta(1 - x_0)
```

Also, should these `\leq` rather than `<`?